### ISSUE TITLE
[web] accessibility announcement does not work reliably on NVDA and JAWS

### ISSUE THREAD

**Issue by yjbanov** [flutter/flutter#116364]
NVDA and JAWS do not reliably interpret the live region that Flutter Web engine injects to produce announcements. The theory is that NVDA and JAWS disagree with other screen readers about how to interpret _new_ live regions. It seems they can only handle preexisting regions that change their contents. If a new live region appears, it is not initially announced.

Possible fix: create live regions - one for polite, and one for assertive - during engine initialization and keep them around. For each announcement, add a `<div>` node into one of the live regions.

**Comment by marcianx** [comment]
To further motivate the change, per https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions, changes within live regions are announced rather than the insertion of live regions themselves.
